<launch>

    <node pkg="rviz" type="rviz" name="rivz" args="-d $(find planner)/config/rviz_config.rviz" />

    <node pkg="tf" type="static_transform_publisher" name="map_to_world" args="0 0 0 0 0 0 1 map world 50" />

    <node pkg="planner" type="global_planning.py" name="global_planning" output="screen">
        <rosparam file="$(find planner)/config/planner_params.yaml" command="load" />
    </node>


</launch>